<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>198</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>277</width>
    <height>198</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>277</width>
    <height>198</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MYCHAT</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>MYCHAT.ico</normaloff>MYCHAT.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_userName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>69</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_rem">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>109</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_auto">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>109</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>自动登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reg">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>139</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>139</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>3</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>密  码：</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
